src/components/expanded-state/ExpandedStateSection.js

```diff
@@ -6,43 +6,75 @@ import { padding } from '@rainbow-me/styles';
 
 const Container = styled(ColumnWithMargins).attrs({
   margin: 12,
-})(({ isNft, isL2 }) => ({
+})(({ isNft, isL2, isTokenHistory }) => ({
   ...padding.object(android ? 19 : 36, 19, 24),
   ...padding.object(android ? 19 : 36, isNft ? 24 : 19, 24),
-  paddingTop: isL2 || isNft ? 24 : android ? 19 : 36,
+  ...padding.object(android ? 19 : 36, isTokenHistory ? 0 : 19, 24),
+  paddingTop: isL2 || isNft || isTokenHistory ? 24 : android ? 19 : 36,
 }));
 
 export default function ExpandedStateSection({
   children,
   isNft,
   title,
+  isTokenHistory,
   ...props
 }) {
   const { colors } = useTheme();
   return (
-    <Container isNft={isNft} {...props}>
-      <Text
-        color={isNft ? colors.whiteLabel : colors.dark}
-        size="large"
-        weight="heavy"
-      >
-        {title}
-      </Text>
-      {typeof children === 'string' ? (
-        <Text
-          color={
-            isNft
-              ? colors.alpha(colors.whiteLabel, 0.5)
-              : colors.alpha(colors.blueGreyDark, 0.5)
-          }
-          lineHeight="big"
-          size="large"
-        >
-          {children}
-        </Text>
+    <>
+      {isTokenHistory ? (
```

Comment:
I think this breaks a bit some high-level concepts like COP or Dependency Inversion Principle. This should be considered as a "higher level" component so should not have "knowledge" about its children's specifications. You should be able to achieve similar things via composition or abstraction. 

On the other hand, looks like `ExpandedStateSection` has two "flows" that are completely separated (when `isTokenHistory` is `true` or `false`. Can't they be just two components?